tests/conftest.py

```diff
@@ -324,3 +329,45 @@ def identity_simple():
     i.provides.add(UserNeed(1))
     i.provides.add(Need(method='system_role', value='any_user'))
     return i
+
+
+@pytest.fixture(scope="module")
+def identity_authenticated():
+    """Simple identity fixture."""
+    identity = Identity(1)
+    identity.provides.add(authenticated_user)
+    return identity
+
+
+@pytest.fixture()
```

Comment:
Minor: Can you check if it's possible to use [these functions](https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/testutils.py) from Invenio-Accounts instead?